### PR TITLE
Bugfix/noid/fix console errors

### PR DESCRIPTION
**Pull request by ma12-co**

### CODE REVIEW

File: src/components/CallView/shared/VideoBottomBar.vue [hunk 1]
```diff
@@ -171,7 +172,19 @@ export default {
 		},
 
 		stopFollowingLabel() {
-			return t('spreed', 'Stop following')
+			return t('spreed', `Stop following`)
```
Comment (danxuliu) [inline src/components/CallView/shared/VideoBottomBar.vue:175]: It should use single quotes instead of backticks.

Comment (nickvergessen) [reply src/components/CallView/shared/VideoBottomBar.vue:175]: So the rebase failed @ma12-co xD

**Comment by nickvergessen** [comment]
Backport in https://github.com/nextcloud/spreed/pull/3697

**Pull request merged**